internal/pkg/mapper/defaultbucket/local_load.go

```diff
@@ -0,0 +1,59 @@
+package defaultbucket
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/orderedmap"
+)
+
+// MapAfterLocalLoad - replace placeholders with default buckets in IM.
+func (m *defaultBucketMapper) MapAfterLocalLoad(recipe *model.LocalLoadRecipe) error {
+	config, ok := recipe.Object.(*model.Config)
+	if !ok {
+		return nil
+	}
+
+	err := m.visitStorageInputTables(config, m.replacePlaceholderWithDefaultBucket)
+	if err != nil {
+		m.Logger.Warnf(`Warning: %s`, err)
+	}
+	return nil
+}
+
+func (m *defaultBucketMapper) replacePlaceholderWithDefaultBucket(config *model.Config, inputTableSource string, inputTable *orderedmap.OrderedMap) error {
+	if !strings.HasPrefix(inputTableSource, "{{:default-bucket:") {
+		return nil
+	}
+
+	sourceWithoutPrefix := strings.TrimPrefix(inputTableSource, "{{:default-bucket:")
+	splitSource := strings.Split(sourceWithoutPrefix, "}}.")
+	if len(splitSource) != 2 {
+		return nil
+	}
+
+	// Get branch
+	branch := m.State.MustGet(config.BranchKey())
+
+	// Get key by path
+	path := filesystem.Join(branch.Path(), splitSource[0])
+	configKeyRaw, found := m.Naming.FindByPath(path)
+	if !found {
+		return fmt.Errorf(`configuration %s contains table "%s" in input mapping referencing to a non-existing configuration`, config.Desc(), inputTableSource)
```

Comment:
`config.Desc()` je v tvare `config "..."`, takze to `configuration` je IMHO naviac. vysledok teraz bude `configuration config ....`.

Ak chces `configuration` namiesto skratky `config`, tak ok, ale uprav to v `ConfigKey.Desc`, nech je to vsade rovnako.

Reply:
Aha ne, tak to jsem si spletl, fixnu